src/components/CreatBlogPost.tsx

```diff
@@ -0,0 +1,144 @@
+import React, {useState} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { 
+    Button, 
+    Box, 
+    Container, 
+    Grid, 
+    TextField, 
+    Typography 
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+    root: {
+        paddingTop: '2rem',
+        paddingBottom: '2rem',
+    },
+
+    title: {
+        textAlign: 'center',
+    },
+    
+    input: {
+        justifyItems: 'left',
+        margin: '2rem'
+    },
+    
+    author: {
+        width: '40%',
+        marginTop: '1rem',
+        marginBottom: '1rem',
+    },
+
+    actionButton: {
+        marginTop: '2rem',
+        display: 'flex',
+        justifyContent: 'space-around',
+        alignItems: 'center'
+    },
+    
+    cancelButton: {
+        padding: '3rem',
+        background: 'red'
+
+    },
+
+    submitButton: {
+        padding: '3rem',
+        paddingLeft: '3.75rem',
+        paddingRight: '3.75rem',
+        background: 'lightGreen'
+    }, 
+
+    textfield: {
+        marginTop: '2rem',
+        marginBottom: '2rem',
+    }
+});
+
+
+
+const CreateBlogPost: React.FC = () => {
+    const classes = useStyles();
+
+    const [post, setPost] = useState(
+        {
+            title: '',
+            author: '',
+            content: ''
+        }
+    );
+
+    const handleSubmit = () => {
+        alert("Title of the Post: " + post.title + "\n" +
+              "Author of the Post: " + post.author + "\n" +
+              "Content of the Post: " + post.content);
+    }
+
```

Comment:
```suggestion
const handleChangeTitle: void = (e: [not sure what type]) => {
setPost((prevPost: Post) => {
  return {
    ...prevPost,
    title: e.target.value
    }
});
```
This isn't 100% correct, but reach out to me for clarification.